torch/_torch_docs.py

```diff
@@ -7995,29 +8009,34 @@ def merge_dicts(*dicts):
            r"""
 tensor(data, *, dtype=None, device=None, requires_grad=False, pin_memory=False) -> Tensor
 
-Constructs a tensor with :attr:`data`.
+Constructs a tensor with no autograd history (a "leaf tensor") by copying :attr:`data`.
```

Comment:
nit: you can link to is_leaf doc if you want to point to a detailed definition of what leaf means
https://pytorch.org/docs/stable/generated/torch.Tensor.is_leaf.html?highlight=leaf#torch.Tensor.is_leaf

Reply:
Good idea

Reply:
Updated to point to the autograd mechanics note, which I think does a better job of explaining leaf tensors